<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenericLexicalEntryForm</class>
 <widget class="QWidget" name="GenericLexicalEntryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>151</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="languageLabel">
     <property name="text">
      <string>Language name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="otherCombo"/>
   </item>
   <item>
    <widget class="LingEdit" name="lineEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LingEdit</class>
   <extends>QLineEdit</extends>
   <header>lingedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
